engine/src/main/java/com/datatorrent/stram/api/StreamingContainerUmbilicalProtocol.java

```diff
@@ -413,7 +413,7 @@ public String toString()
    * @param operators
    * @param msg
    */
-  void reportError(String containerId, int[] operators, String msg);
+  void reportError(String containerId, int[] operators, String string, String logFileName, long logFileOffset);
```

Comment:
where is the logic that if the new rollingFileAppender is not used, do not emit the filename and offset in the events?